### PR TITLE
New marketarea records for the US

### PR DESCRIPTION
**Pull request by stepps00**
Related to #235.

This PR adds new `marketarea` records for the United States. The geometries are based on the Nielsen DMAs, found here: https://www.thevab.com/wp-content/uploads/2018/05/2017-2018-TV-DMA-Map.pdf. 

Geometries represent the areas as of 2018 and are mostly based off of existing `county` geometries, except in:

- Solono County (split W/E)
- Palm Springs (distinct cut-out)
- Tahoe area (split)
- Utica (split W/E)

Noting that:
- All geometries have been validated
- Alt names have been stored in the appropriate `name:*` property
- All Nielsen codes/identifiers have been stored as a concordance
- Hierarchies have all been set to include the US, the USA empire, and North America
- Hierarchies and parent_id values are controlled
- All records are tagged as current

### CODE REVIEW

**Comment by stepps00** [comment]
These issues are related and should be addressed before merge:
- https://github.com/whosonfirst/whosonfirst-placetypes/issues/17
- https://github.com/whosonfirst/whosonfirst-sources/issues/138

File: data/136/066/506/9/1360665069.geojson [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "geom:area_square_m":39257924550.779366,
+    "geom:bbox":"-101.183997,31.409778,-98.474685,33.83608",
+    "geom:latitude":32.507712,
+    "geom:longitude":-99.8001,
+    "iso:country":"US",
+    "lbl:latitude":32.27626,
+    "lbl:longitude":-99.574181,
+    "mps:latitude":32.27626,
+    "mps:longitude":-99.574181,
+    "mz:hierarchy_label":1,
```
Comment (nvkelso) [inline data/136/066/506/9/1360665069.geojson]: Please set the hierarchy_label to `0` as these aren't useful to include / aren't referenced colloquially.

**Comment by nvkelso** [comment]
A few like data/136/066/527/3/1360665273.geojson should have both _name + state_ and _name_ alone in the preferred and variant namespaces?

**Comment by nvkelso** [comment]
@stepps00 These generally look good but let's have another look at them together on your screen to make sure the great lakes were processed inclusive of the water. Hard to find those in the diff here.

**Comment by stepps00** [comment]
https://github.com/whosonfirst-data/whosonfirst-data/pull/1328#issuecomment-425319498 taken care of in 8bb5246.

**Pull request merged**